### ISSUE TITLE
Use 5 letter radar codes and drop "country" argument in download_vpfiles() and select_vpfiles()

### ISSUE THREAD

**Issue by peterdesmet** [adokter/bioRad#173]
`download_vpfiles()` currently requires both the argument `radar` and `country` to be provided. But it combines them in a way that leads to non-existing radars and thus files that cannot be downloaded:

```R
download_vpfiles(
  date_min = "2016-10-01",
  date_max = "2016-12-31",
  country = c("be", "se"),
  radar = c("jab", "wid", "ang"),
  directory = "my_data"
)
```

Will search for:

```
bejab: ok
sejab: error
bewid: ok
sewid: error
beang: error
seang: ok
```

Given that **both arguments are required anyway** (radar without country does not work and vice versa) and to avoid against creating radar codes that do not exist, I would argue to drop the `country` parameter, and let the user explicitly provide 5 letter radar codes in `radar`:

```R
download_vpfiles(
  date_min = "2016-10-01",
  date_max = "2016-12-31",
  radar = c("bejab", "bewid", "seang"),
  directory = "my_data"
)
```

---

For the parallel function `select_vpfiles()`

```R
select_vpfiles(
  date_min = "2016-10-01",
  date_max = "2016-12-31",
#  country = c("be"),
  radar = c("wid"),
  directory = "my_data"
)
```

The radar and country ARE optional, i.e. one can correctly search for dates, dates + radars, dates + countries or all 3. Still, I would opt here too to drop the country parameter and ask for 5 letter radar codes, because:

- 3 letter radar codes _can_ be shared between countries, leading to unexpected results. Better if the user is explicit
- I think most analyses want to be explicit about the radars they select
- That way we can keep `download_vpfiles()` and `select_vpfiles()` similar (e.g. you can reuse a radar code list to download and select)

I don't think this will break current functionality. @adokter @stijnvanhoey @CeciliaNilsson709 comments?

**Comment by stijnvanhoey** [comment]
I agree, should indeed not break anything (just taing care of backward compatibility).

**Comment by peterdesmet** [comment]
@stijnvanhoey question about arguments and positions:

Currently:

```R
download_vpfiles(date_min, date_max, country, radar, directory = ".")
select_vpfiles(directory, date_min, date_max, country = NULL, radar = NULL)
```

Is it OK to reorder and rename (for radar**s**) these arguments to:

```R
download_vpfiles(directory = ".", date_min, date_max, radars, overwrite = FALSE)
select_vpfiles(directory, date_min, date_max, radars)
```

Or would you do this differently?

**Comment by peterdesmet** [comment]
@stijnvanhoey see #176: I opted to move `directory` to last parameter in `select_vpfiles()` (all parameters are optional there), so non-optional arguments for `download_vpfiles()` can remain at the beginning.

**Comment by peterdesmet** [comment]
This was implemented in #176 in 2018 and was included in bioRad 0.4.0.
